src/datasets/utils/info_utils.py

```diff
@@ -9,6 +10,28 @@
 logger = get_logger(__name__)
 
 
+class VerificationMode(enum.Enum):
+    """`Enum` that specifies which verification checks to run.
+
+    The default mode is `BASIC`, which will perform only rudimentary checks to avoid slowdowns
+    when generating/downloading a dataset for the first time.
+
+    The verification modes:
+
+    |                    | Verification checks                                                           |
+    |--------------------|------------------------------------------------------------------------------ |
+    | `FULL`             | Split checks, uniqueness of the keys yielded in case of the GeneratorBuilder  |
```

Comment:
is uniqueness of the keys check depended on the verification_mode?

Reply:
Yes, it does

Reply:
can you please point me to the place in the code? I can't find it.... 🙈

Reply:
This is the commit in which this check was introduced: https://github.com/huggingface/datasets/commit/beca0840a03f45f6089ff26e5776e05916a5be96